### PR TITLE
Switch to new CAS service account structure

### PR DESCRIPTION
**Pull request by robertauer**

### CODE REVIEW

**Comment by nhinze23** [comment]
Local volume ist missing and the modification of the `doguctl state` statements.

**Review by nhinze23 — CHANGES_REQUESTED** [review]
Local volume ist missing and the modification of the `doguctl state` statements.

**Comment by robertauer** [comment]
> Local volume ist missing and the modification of the `doguctl state` statements.

I think this is done in https://github.com/cloudogu/redmine/pull/129

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library(['github.com/cloudogu/ces-build-lib@2.2.1', 'github.com/cloudogu/dogu-build-lib@v2.3.1']) _
+@Library(['github.com/cloudogu/ces-build-lib@2.3.0', 'github.com/cloudogu/dogu-build-lib@v2.3.1']) _
```
Comment (jelemux) [inline Jenkinsfile:2]:
```suggestion
@Library(['github.com/cloudogu/ces-build-lib@2.4.0', 'github.com/cloudogu/dogu-build-lib@v2.4.0']) _
```

**Comment by jelemux** [comment]
> I think this is done in #129

Yes, there was this old attempt to do this but I think, we recognized then that these changes in CAS were necessary for it to work. We have a separate story for that (49632 in our internal tracker). You could, of course, implement this directly here as well. Otherwise, testing this in a multinode environment won't be possible.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 import com.cloudogu.ces.cesbuildlib.*
 import com.cloudogu.ces.dogubuildlib.*
 import com.cloudogu.ces.zaleniumbuildlib.*
```
Comment (jelemux) [inline Jenkinsfile:5]: Do we really need zaleniumbuildlib here?

**Review by jelemux — CHANGES_REQUESTED** [review]
Some minor nitpicks and maybe do consider adding the local volume and the modification of the doguctl state statements here again, as, like I've said, we won't be able to test this against multinode, otherwise.

File: Jenkinsfile (hunk 1 above)
Comment (robertauer) [reply Jenkinsfile]: Done

File: Jenkinsfile (hunk 2 above)
Comment (robertauer) [reply Jenkinsfile:5]: No, this statement doesn't seem to make sense as the library is not imported at the top. I'll delete it.

**Pull request merged**